Yoast/javascript
# Update rules for non-passives and exception list

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Expands list of exceptions for words that look like passive voice forms but are not and adds a rule that checks for those exceptions.
* [Yoast SEO Premium] Improves the recognition of passive voice (including for non-passives that are spelled like passives).

## Test instructions for team lingo

This PR can be tested by following these steps:

* Run `yarn test` and make sure all tests still pass.
* In the content-analysis app:
  * Set the locale to id_ID;
  * Paste [this text](https://docs.google.com/document/d/1oShcLuQLscN1WCLbjDk_jLOqgQvJEw24wz-aCmZMsV4/edit) in the Text input window (it won't detect any passives);
  * Add these sentences containing Indonesian passive verb forms: `Dina merupakan anak didikan Pak Hasan. Dikau selalu pergi ke disko tiap malam. Suaminya sedang dinas ke luar kota.`
  * The passive voice assessment should read `You are using enough active voice. Great!`


## QA Test instructions

This PR can be tested by following these steps:

* Set the WP environment language to Indonesian
* Test this functionality in Wordpress SEO Free
* You should see a bullet assessment for passive verb forms in the Readability section
* Add these sentences to your post:
`Dia melanjutkan kuliahnya di Jakarta. Rumah yang dindingnya berwarna hijau itu roboh. Disiplinlah dalam menuntut ilmu! Dia duduk di atas dipan.`
* Check that the passive voice bullet turns red


## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:

## UI changes

## Quality assurance


Fixes #https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-575
